src/nvim/screen.c

```diff
@@ -4507,9 +4507,17 @@ void status_redraw_all(void)
  * mark all status lines of the current buffer for redraw
  */
 void status_redraw_curbuf(void)
+{
+  status_redraw_buf(curbuf);
+}
+
+/*
+ * mark all status lines of the specified buffer for redraw
+ */
```

Comment:
```
src/nvim/screen.c:4514:  /*-style comment found, it should be replaced with //-style.  /*-style comments are only allowed inside macros.  Note that you should not use /*-style comments to document macros itself, use doxygen-style comments for this.  [readability/old_style_comment] [5]
```

Reply:
That's what I was referring to in my comment above. I was attempting to minimize the changes from the function this was based on, to simplify review.

Will fix.

Reply:
We use the lint warning as a means to gradually convert the code base to Neovim's coding style.  Usually, a separate commit with the simple commit message "lint" is sufficient to address those.